src/ape_accounts/_cli.py

```diff
@@ -86,17 +87,38 @@ def generate(cli_ctx, alias):
 
 
 # Different name because `import` is a keyword
-@cli.command(name="import", short_help="Add a new keyfile account by entering a private key")
+@cli.command(
+    name="import", short_help="Add a new keyfile account by entering a private key or seed phrase"
+)
+@click.option(
+    "--use-mnemonic", "import_from_mnemonic", help="Import a key from a mnemonic", is_flag=True
+)
+@click.option(
+    "--hdpath",
```

Comment:
In `ape-trezor`, it is `--hd-path` (with extra hyphen).
We should decide on one and make them consistent.
Perhaps use the same as `ape-trezor` for now.
https://github.com/ApeWorX/ape-trezor/blob/main/ape_trezor/_cli.py#L53

Reply:
ok great, fixed this in latest commit :)